storage/pebble/open.go

```diff
@@ -21,7 +20,7 @@ func NewBootstrappedRegistersWithPath(dir string) (*Registers, *pebble.DB, error
 	if err != nil {
 		return nil, nil, fmt.Errorf("failed to initialize pebble db: %w", err)
 	}
-	registers, err := NewRegisters(db)
+	registers, err := NewRegisters(db, 0)
```

Comment:
instead of hardcoding `0` everywhere, let's make this a const. something like `pruner.Disabled`